RadarSDK/Radar.h

```diff
@@ -396,6 +396,15 @@ typedef void(^ _Nonnull RadarIPGeocodeCompletionHandler)(RadarStatus status, Rad
  */
 + (void)ipGeocode:(RadarIPGeocodeCompletionHandler)completionHandler;
 
```

Comment:
nit: maybe it's better to revise the name, so the difference between `ipGeocode:` and `ipGeocode:completionHandler` is more obvious?

Reply:
agreed, good call, revised the naming to `geocodeDeviceIP` vs `geocodeIP` -- thoughts?